inductiva/utils/interpolation.py

```diff
@@ -1,51 +1,61 @@
 """Interpolation utilities."""
 
-from typing import Tuple
+from typing import Literal, Optional, Tuple
 
 import numpy as np
 import scipy
 
 
-def interpolate_to_uniform_grid(x: Tuple[np.ndarray],
-                                values: np.ndarray,
-                                x_grid: Tuple[np.ndarray],
-                                threshold_distance=None):
-    """Interpolate values to a uniform grid.
+def interpolate_to_coordinates(
+    x: Tuple[np.ndarray],
```

Comment:
This is a very weird name, since in other cases we use x,y. What do you think of the more explicity `coordinates` and `coordinates_interpolation`?